api_docgen/Makefile

```diff
@@ -27,5 +27,9 @@ odoc-%:
 ocamldoc-%:
 	$(MAKE) -C ocamldoc $* ROOTDIR=../..
 
+.PHONY: clean
 clean:
 	rm -rf build odoc/build ocamldoc/build
+
+.PHONY: distclean
+distclean: clean
```

Comment:
I've not 100% convinced myself that these shouldn't be empty recipes (either with a single `@` command below or ending with a semi-colon), but I think it's OK for a PHONY target.